free-science-books.md

```diff
@@ -140,6 +140,7 @@
 * [From Vitamins to Baked Goods: Real Applications of Organic Chemistry (2012)](https://unglue.it/work/145870/) - Kieran Gallagher and Lizhi Fan
 * [Green Chemistry](https://unglue.it/work/488195/) - Mazaahir Kidwai, Neeraj Kumar Mishra
 * [Introduction to Chemistry](https://unglue.it/download_ebook/54777/) - Tracy Poulsen (PDF)
+* [Structural Biochemistry](https://en.wikibooks.org/wiki/Structural_Biochemistry) - Wikibooks 2012
```

Comment:
Good question now looking at it. It was published in 2012 reference was from wiki books, I guess though this could be removed or placed within brackets like ones before it inside the name section.

Reply:
What do you advise? Removal of the year or should I place the year within the name section after the name? 👍

Reply:
I’ve chosen to remove the year, as it’s not needed.